NEasyAuthMiddleware/Core/EasyAuthOptions.cs

```diff
@@ -6,6 +6,7 @@ namespace NEasyAuthMiddleware.Core
 {
     public class EasyAuthOptions : AuthenticationSchemeOptions
     {
+        public string Provider { get; set; } = "aad";
```

Comment:
Might need to introduce an enum here

Reply:
Or at least another class which has known values.

`provider = KnownProviders.AzureActiveDirectory`

Reply:
That's a good idea, incoming!

Reply:
I've updated the implementation and docs. I opted for `KnownAuthProviders` to match nomenclature used in your existing constant classes but let me know if you prefer something else!

NOTE: Users can create OAuth connections to custom providers, so using an enum didn't feel right since the provider string could be something totally custom. That's why I opted for constant strings vs an enum.
![image](https://github.com/user-attachments/assets/5b45a7e5-cf9a-4810-9bd9-45b8f043d5cd)